scripts/main/sidebar.js

```diff
@@ -137,7 +137,7 @@ sidebar.toggle = function (is_user_initiated) {
 		header.dom(".button--info").toggleClass("active");
 		lychee.content.toggleClass("content--sidebar");
 		lychee.imageview.toggleClass("image--sidebar");
-		if (typeof view !== "undefined") view.album.content.justify(album.json ? album.json.photos : []);
```

Comment:
In case you were wondering "what the hell was that for", it was to prevent the standalone view mode from crashing, I believe. But with #323 I guess that may no longer be needed?

Reply:
Thanks good to know.